libvirt/domain.go

```diff
@@ -252,6 +291,15 @@ func setCoreOSIgnition(d *schema.ResourceData, domainDef *libvirtxml.Domain) err
 	return nil
 }
 
+func setCoreOSIgnition(d *schema.ResourceData, domainDef *libvirtxml.Domain) error {
```

Comment:
I don't quite get it. You're adding another function with the same name as above? Who calls it?

Reply:
sorry, I uploaded the wrong stuff, I will re-upload this soon, thanks for pointing out